R/compare.R

```diff
@@ -56,9 +56,43 @@
 #' @param ignore_encoding Ignore string encoding? `TRUE` by default, because
 #'   this is R's default behaviour. Use `FALSE` when specifically concerned
 #'   with the encoding, not just the value of the string.
+#' @param ignore_name_order Ignore changes to the order of named components in
+#'   lists or vectors.
+#' @param ignore_NULLs In lists and pairlists, ignore elements that are set to
+#'   `NULL`.
+#' @param ignore_private Components matching the patterns in this vector are
```

Comment:
What do you think of `ignore_private_paths` as a name? I had initially just assumed this applied to attributes.

Reply:
Sure.  

PS:  I don't really know the expected workflow, such as clicking on "Resolve conversation", etc

Reply:
If you're confident the conversation is resolved and your last response is short, feel free to resolve it. Otherwise it's slightly better to leave it for me to resolve since that'll make it more likely for me to read the discussion.

But hold off on making any more changes for now; I want to let these ideas sink in a bit more, and maybe pull off some pieces into separate PRs. (e.g. the `list_as_dictionary` changes are independent, so could be done independently of the options.)